### PR TITLE
DM-14842: Fix deprecation warnings from PropertyList/Set.get

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/synpipe/compareModel.py [hunk 1]
```diff
@@ -103,8 +100,7 @@ def main(root, visit, ccds, galType='sersic', output='outputs/'):
 
     npTable = writeNumpyTable(fakeTable)
     rerunName = root.split('/')[-2]
-    fitsTable = astropy.table.Table(npTable).write(output+'/'+rerunName +
-                                                   '_galMags.txt', format='ascii')
+    astropy.table.Table(npTable).write(output+'/'+rerunName + '_galMags.txt', format='ascii')
```
Comment (yalsayyad) [inline python/lsst/synpipe/compareModel.py]: Add some extra spaces around the  + while you're already editing this line. (Looks like it's not  approaching 110)

File: python/lsst/synpipe/makeBlendedCat.py [hunk 2]
```diff
@@ -1,10 +1,5 @@
-#!/usr/bin/env python
```
Comment (yalsayyad) [inline python/lsst/synpipe/makeBlendedCat.py:1]: Why does this go away and not in `python/lsst/synpipe/makeFakeGalaxy.py` too?

Comment (r-owen) [reply python/lsst/synpipe/makeBlendedCat.py:1]: I must have missed that one. I'll fix it and a bunch of others.

Note that many of these same modules have `if __name__ == "__main__":` sections, which don't belong in library code. But I'm leaving them for now.

**Comment by r-owen** [comment]
I fixed the two issues you raised

**Review by yalsayyad — APPROVED** [review]

**Pull request merged**